Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/calculation/DccWalletInfoCalculationManager.kt

```diff
@@ -20,29 +22,44 @@ class DccWalletInfoCalculationManager @Inject constructor(
     private val timeStamper: TimeStamper,
 ) {
 
-    suspend fun triggerCalculation(
-        configurationChanged: Boolean = true
-    ) {
-        Timber.d("triggerCalculation()")
-        val now = timeStamper.nowUTC
+    /**
+     * Trigger [DccWalletInfo] calculation for all persons
+     */
+    suspend fun triggerCalculation(configurationChanged: Boolean = true) {
         initCalculation()
-        personCertificatesProvider.personCertificates.first().forEach {
+        val persons = personCertificatesProvider.personCertificates.first()
```

Comment:
We should not mix here. This is only calculation. Cleanup should be a separate method in a different class

Reply:
done  see `DccWalletInfoCleaner`